src/docs/tutorials/IntroductionToMarkdown.md

```diff
@@ -268,15 +283,15 @@ You can nest them:
 >
 > - You can quote a list.
 > - Etc.
-```
+```</code></pre>
 
 ### Code Spans
 
-```bash
+<pre><code>```
 - `<code>` spans are delimited by backticks.
```

Comment:
This will does not work pls replace `<code>` with `&#60;code&#62;`.

Reply:
Not changed, not resolved?

Reply:
Eh, I just did. Wait.